app/models/participant_profile/npq.rb

```diff
@@ -37,6 +37,10 @@ def fundable?
       npq_application&.eligible_for_dfe_funding
     end
 
+    def schedule_for(*)
+      schedule
+    end
```

Comment:
i want to have this duck type-able so the interface remains the same across both. NPQ will at some point will get something like induction/training records to have parity with ECF in which case this can be dropped and the implementation can be pushed up to the superclass.